pydevd_plugins/extensions/pydevd_plugin_omegaconf.py

```diff
@@ -0,0 +1,114 @@
+# based on https://github.com/fabioz/PyDev.Debugger/tree/main/pydevd_plugins/extensions
+import os
+import sys
+from functools import lru_cache
+from typing import Any, Dict, Sequence
+
+from _pydevd_bundle.pydevd_extension_api import (  # type: ignore
+    StrPresentationProvider,
+    TypeResolveProvider,
+)
+
+
+@lru_cache(maxsize=128)
+def find_mod_attr(mod_name: str, attr: str) -> Any:
+    mod = sys.modules.get(mod_name)
+    return getattr(mod, attr, None)
+
+
+class OmegaConfDeveloperResolver(object):
+    def can_provide(self, type_object: Any, type_name: str) -> bool:
+        Node = find_mod_attr("omegaconf", "Node")
+        return Node is not None and issubclass(type_object, Node)
+
+    def resolve(self, obj: Any, attribute: str) -> Any:
+        return getattr(obj, attribute)
+
+    def get_dictionary(self, obj: Any) -> Any:
+        return obj.__dict__
+
+
+class OmegaConfUserResolver(StrPresentationProvider):  # type: ignore
+    def can_provide(self, type_object: Any, type_name: str) -> bool:
+        Node = find_mod_attr("omegaconf", "Node")
+        return Node is not None and issubclass(type_object, Node)
+
+    def resolve(self, obj: Any, attribute: Any) -> Any:
+        if isinstance(obj, Sequence):
+            if isinstance(attribute, str):
+                attribute = int(attribute)
+            val = obj.__dict__["_content"][attribute]
+        else:
+            val = obj.__dict__["_content"][attribute]
+
+        return val
+
+    def _is_simple_value(self, val: Any) -> bool:
+        ValueNode = find_mod_attr("omegaconf", "ValueNode")
+        return (
+            isinstance(val, ValueNode)
+            and not val._is_missing()
+            and not val._is_interpolation()
+            and not val._is_none()
```

Comment:
Unlikely to matter much, but testing `_is_none()` first is more efficient
```suggestion
            and not val._is_none()
            and not val._is_missing()
            and not val._is_interpolation()
```